LandSandBoat/server
# Search by lobbyview FD if it exists

**_I affirm:_**
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

On logout with multiple clients on the same IP, previous behavior was to default to the first matching IP.

lobbydata after login stores the descriptor for the lobbyview, so search this first before falling back to IP.

Test case used for verification:
1. Login with Client A
2. Login with Client B
3. Logout with Client B
4. Select Char with Client B

Previous version:
1. Client B would show Client A characters

Post-PR:
1. Client B shows Client B characters (expected)
